.github/ISSUE_TEMPLATE/tutorials.yaml

```diff
@@ -82,3 +82,7 @@ body:
   - type: markdown
     attributes:
       value: Thank you for your feedback! If you are interested in joining the IBM Quantum Feedback Program, <a href="https://www.ibm.com/quantum/feedback-program">sign up here</a>.
+
+  - type: markdown
+    attributes:
+      value: Tagging the [Docs content team](https://github.com/orgs/Qiskit/teams/docs-content-team)
```

Comment:
I don't understand this part?

Reply:
It's a test in case the team tag doesn't work. Curious if this will properly tag us if the team is linked in the issue body

Reply:
I doubt this would give a notification. It'd only link to the page. You'd want to use `@` instead.